src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsintegrationapi/services/GetSentencesForPersonServiceTest.kt

```diff
@@ -93,13 +118,29 @@ internal class GetSentencesForPersonServiceTest(
       whenever(nDeliusGateway.getSentencesForPerson(nDeliusCRN)).thenReturn(
         Response(
           data = listOf(
-            Sentence(
+            IntegrationApiSentence(
               dateOfSentencing = LocalDate.parse("2003-01-01"),
               isActive = true,
+              terms = listOf(
+                IntegrationApiTerm(
+                  years = 4,
+                ),
+                IntegrationApiTerm(
+                  weeks = 12,
+                ),
+              ),
             ),
-            Sentence(
+            IntegrationApiSentence(
               dateOfSentencing = LocalDate.parse("2004-01-01"),
               isActive = false,
+              terms = listOf(
+                IntegrationApiTerm(
+                  weeks = 18,
+                ),
+                IntegrationApiTerm(
+                  months = 11,
+                ),
+              ),
```

Comment:
Why are you returning IntegrationApiSentence models whenever you are getting sentence information from nDelius? Surely you should be responding with an nDelius model constructor?